clouddriver-ecs/src/integration/resources/testartifacts/createServerGroup-artifact-Fargate-targetGroup-artifactFile.json

```diff
@@ -0,0 +1,42 @@
+
+{
+  "family": "PLACEHOLDER",
+  "containerDefinitions": [
+    {
+      "name": "application",
+      "image": "PLACEHOLDER",
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "spinnaker-ecs-demo",
+          "awslogs-region": "us-west-2",
+          "awslogs-stream-prefix": "spinnaker"
+        }
+      },
+      "portMappings": [
+        {
+          "hostPort": 80,
+          "protocol": "tcp",
+          "containerPort": 80
+        }
+      ],
+      "environment": [
+        {
+          "name": "PORT",
+          "value": "80"
+        }
+      ],
+      "cpu": 256,
+      "memoryReservation": 512,
+      "essential": true
+    }
+  ],
+  "cpu": "256",
+  "memory": "512",
+  "requiresCompatibilities": [
+    "FARGATE"
+  ],
+  "executionRoleArn": "arn:aws:ecs:::task-definition/SpinnakerManaged:1",
+  "networkMode": "aws-vpc",
+  "taskRoleArn" : "arn:aws:ecs:::task-definition/PLACEHOLDER:1"
```

Comment:
for both roles (and in general for all resource types), please make the value look like an actual IAM ARN (e.g., `arn:aws:iam:::role/testTaskRole`) or an obviously-named tests value (e.g., `arn:testRole`). using the ARN format for a different kind of resource is confusing, esp. in assertions (context: I discovered this b/c above we are asserting "executionRoleArn" equal "...task-definition..." ?)